frontend/src/components/home/midsection/mid.css

```diff
@@ -1,18 +1,7 @@
-.bottom-text {
```

Comment:
it looks like you have declared the same font-family several times, I would probably use the `*` css selector at the top of the file and just declare the `Medium` type of font for the elements you want as Medium